public/app/features/datasources/components/picker/DataSourceList.tsx

```diff
@@ -98,7 +99,9 @@ function EmptyState({ className, onClickCTA }: { className?: string; onClickCTA?
   const styles = useStyles2(getEmptyStateStyles);
   return (
     <div className={cx(className, styles.container)}>
-      <p className={styles.message}>No data sources found</p>
+      <p className={styles.message}>
+        <Trans i18nKey="data-source-picker.list.no-data-source-message"> No data sources found</Trans>
```

Comment:
```suggestion
        <Trans i18nKey="data-source-picker.list.no-data-source-message">No data sources found</Trans>
```

extra space is probably not needed